### DISCUSSION TITLE
Virtual Folder Structure?

### DISCUSSION THREAD

**Discussion by ElliotHYLee** [pocketbase/pocketbase#2573] [Q&A]
This may be super minor question.

Currently I'm using  prefix for each collection name in the admin panel so I can kind of group the collections visually. i.e "prd_oders", "prd_paymens" where "prd" stands for product or so.

Do you have any plan or so to introduce a folder strucutre to the collection? I think the the "folder strcutre" can be a totally virtual, just a namespace to group the collectins instead of actual url or any.

I just thought it could be a good-to-have feature in the future if applicable.

**Comment by ganigeorgiev** [comment]
This was requested before (eg. https://github.com/pocketbase/pocketbase/discussions/2405), it exists in my todo but for now is left aside.
